site/components/case-study/style.css

```diff
@@ -0,0 +1,126 @@
+@value black, white, linesOnBlack from "../../css/_colors.css";
+@value mediumScreen, largeScreen from "../../css/_breakpoints.css";
+
+.heading {
+  composes: fontL from "../../css/typography/_fonts.css";
+  max-width: 1000px;
```

Comment:
please discuss with a designer what the max width for all elements on the homepage should be - we are using `1000px` in some places and `1080px` in others